db/seeds.rb

```diff
@@ -1,27 +1,27 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 
-
-# GENERATE SUCCESS STORY Content
-load './db/seeds/success_stories.rb'
-
 # ******* NOTE ********
-# You will have problems if you try to change the titles of courses/sections/lessons, since that's currently what's used to uniquely identify them!
-# *********************
+# You will have problems if you try to change the titles of 
+# courses/sections/lessons, since that's currently what's used to uniquely 
+# identify them!
+
 
 # The Structure:
 # Course Has Many Sections. Section Belongs To Course.
 # Section Has Many Lessons. Lesson Belongs To Section.
 
-# with these values it prevents a course, section or lesson being assigned the same value, avoiding conflict
-course_position = 0
-section_position = 0
-lesson_counter = 0
+# Prevent conflict of a course, section or lesson being assigned the same value
+@course_position = 0
+@section_position = 0
+@lesson_counter = 0
 
-# number just throw all the positions into the stratosphere to avoid the annoyance of having to not duplicate them when updating lessons
+# High number to just throw all the positions into the stratosphere to avoid 
+# the annoyance of having to not duplicate them when updating lessons
 incrementer = 2000
 
-# Public: Only run this update attributes if all have one or more records in the database
+# Public: Only run this update attributes if all have one or more records
+# in thedatabase
 if Course.all.any? && Section.all.any? && Lesson.all.any?
```

Comment:
We might not need this now that the courses are in their own files. We can hard code the position of the courses and sections in each course and have a local incriminator for the lessons in each course file. 

I never really understood why this was here in the first place anyway, it's causes more problems than its solved when I've personally been running the seeds file.

Reply:
Just remembered what this is for 🙈 
We have a unique constraint on the position column, so if we add a new lesson into the middle of a section/course it will blow up because the position won't be unique.

We can still hard code the course and section positions because theres so few of them. But we should keep the `@lesson` incriminator. Can you change this particular bit of code to this please?

```
if Lesson.any?
  Lesson.all.each { |l| l.update_attribute(:position, l.position + incrementer) }
end
```